login.py

```diff
@@ -0,0 +1,32 @@
+import hashlib
+from datetime import datetime, timedelta
+
+import jwt
+from flask import request, make_response, jsonify, Blueprint
+from werkzeug.exceptions import Unauthorized
+
+from tables import User
+
+login_blueprint = Blueprint('login', __name__, url_prefix='/login')
+
+
+def decode_token(token):
+    try:
+        return jwt.decode(token, 'very_important_secret', algorithms=['HS256'])
```

Comment:
@YanaRudyi such literals as decoding algorithm it is always a better idea to move to the configuration file to be changed easily from environment variables (other settings sources) if required